packages/e2e/test/web-extension/extension/ui.html

```diff
@@ -17,7 +17,8 @@ <h1>Wallet</h1>
   </div>
   <h2>ADA price</h2>
   <div id="adaPrice">Loading...</div>
-  <button id="createKeyAgent">Create KeyAgent</button>
+  <button id="createWallet1">Create Wallet 1</button>
+  <button id="createWallet2">Create Wallet 2</button>
```

Comment:
```suggestion
  <button id="activateWallet1">Activate Wallet 1</button>
  <button id="activateWallet2">Activate Wallet 2</button>
  <button id="destroyWallets">Destroy Wallets</button>
```
Let's add an example of deactivating and clearing the stores too

Reply:
added here: [deb1c66](https://github.com/input-output-hk/cardano-js-sdk/pull/502/commits/deb1c6617b48ac03ce5a59e0065db69a114cebee)